pkg/app/web/src/components/settings-page/piped/components/piped-table-row.tsx

```diff
@@ -131,6 +138,17 @@ export const PipedTableRow: FC<Props> = memo(function PipedTableRow({
     onDisable(pipedId);
   }, [pipedId, onDisable]);
 
+  const connectionStatusColor = useCallback((status: Piped.ConnectionStatus) => {
+    switch (status) {
+      case Piped.ConnectionStatus.UNKNOWN:
+        return "secondary";
+      case Piped.ConnectionStatus.ONLINE:
+        return "primary";
```

Comment:
I feel like Green looks more right to me but what do you think?
```suggestion
        return "success";
```

Reply:
It's. but the Badge material does not provide that color type so we can't use that 😢 
![Screen Shot 2021-12-07 at 16 46 31](https://user-images.githubusercontent.com/32532742/144987717-fc31a4bb-2101-4530-8757-f73d0a99f940.png)

Reply:
You're right! Looks like the color "success" got introduced from v5: https://mui.com/api/badge/#props

Reply:
I'm thinking about upgrading material-ui/core to v5, we're going to face a couple of issues to do so though.

Reply:
Well, time to update our web dependencies 😂  lets me address it later 👍

Reply:
Alright. For the color, let's change to it after that.